12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,5 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+lst = []
+ascii = range(97, 123)
+for i in ascii:
+     lst.append([ x for x in chr(i)])
+print lst
```

Comment:
That's not really a list comprehension. Try getting rid of the 'for' loop and move the whole thing inside the [].
Also the variable name ascii isn't really correct  all the characters between 0-127 are ascii. What we have here is maybe lower_case_letters. or ascii_of_lower_case_letter.